packages/visx-xychart/src/utils/findNearestDatumX.ts

```diff
@@ -0,0 +1,37 @@
+import { AxisScale } from '@visx/axis';
+import findNearestDatumSingleDimension from './findNearestDatumSingleDimension';
+import { NearestDatumArgs } from '../types';
+
+export default function findNearestDatumX<
+  XScale extends AxisScale,
+  YScale extends AxisScale,
+  Datum extends object
+>({
+  xScale: scale,
+  xAccessor: accessor,
+  svgCoords,
```

Comment:
This utility seems useful beyond `svg`. Perhaps remove the `svg` from naming at least, or what about `point`?

Reply:
great suggestion! done.